### ISSUE TITLE
<igx-grid> Inline editable combobox missing label

### ISSUE THREAD

**Issue by GibletJr** [IgniteUI/igniteui-angular#11076]
### Description  
When entering edit mode for a cell that utilizes a combobox/drop-down, FastPass generates a "**label**: Ensures every form element has a label" alert

 * igniteui-angular version: 12.3.1
 * browser: Microsoft Edge Version 98.0.1108.55 (Official build) (64-bit)

### Steps to reproduce  

1. Navigate to https://www.infragistics.com/angular-demos/grid/grid-editing
2. Trigger Edit Mode by double-clicking within a cell in the **Available At** column
3. Click on down arrow to open options list
4. Run FastPass analysis

### Result  
**Path**
.igx-input-group--fluent > .igx-input-group__bundle > .igx-input-group__bundle-main > input[name="comboInput"][readonly=""][type="text"]
**Snippet**
<input igxinput="" name="comboInput" type="text" readonly="" aria-required="false" class="igx-input-group__input">
**How to fix**
Fix ONE of the following:
Form element does not have an implicit (wrapped) <label>
Form element does not have an explicit <label>
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute
Element has no placeholder attribute
Element's default semantics were not overridden with role="none" or role="presentation"

### Expected result  
No FastPass alerts

### Attachments  
Attach a
![FastPass - Missing Label Alert Screenshot](https://user-images.githubusercontent.com/56270402/155243753-3a3c3cf9-c18a-405b-b682-f5bb3dd26a6f.jpg)
 sample if available, and screenshots, if applicable.

**Comment by GibletJr** [comment]
@Lipata - do you have an ETA on when this bug will be resolved?

**Comment by Lipata** [comment]
@GibletJr thank you for the suggestions. We will consider them and will write you how and when we can proceed.

**Comment by GibletJr** [comment]
@Lipata Checking back in on ETA.

**Comment by Lipata** [comment]
Hey, @GibletJr. I think we can use some of your suggestions but will need to implement and test them with the different accessibility tools. To be honest we never did this against the tool you are using - FastPass. @radomirchev I think we can work on this along with #11077, when we have a dedicated sprint for bug fixes and improvements.

**Comment by GibletJr** [comment]
@Lipata / @radomirchev - could you please provide an estimate on when the next dedicated sprint for bug fixes and improvements is expected to occur?

**Comment by Lipata** [comment]
@GibletJr, we have put this task in our next sprint, which is starting on the 23rd of May.

**Comment by GibletJr** [comment]
> @GibletJr, we have put this task in our next sprint, which is starting on the 23rd of May.

@Lipata - Thanks for the timeline update. Any chance that #11077 can also be addressed in the next sprint as well?

**Comment by GibletJr** [comment]
@PlamenaMiteva - could you provide more specificity to your recommendation by possibly providing the solution based on the linked demo - [https://www.infragistics.com/angular-demos/grid/grid-editing](https://www.infragistics.com/angular-demos/grid/grid-editing)

**Comment by PlamenaMiteva** [comment]
@GibletJr, this is the code for the "Available At" column. You can set the combo's placeholder in the following way:

`<igx-column field="Locations" header="Available At" [editable]="true" [filterable]="false" width="220px">
                <ng-template igxCell let-cell="cell">
                    {{ parseArray(cell.value) }}
                </ng-template>
                <ng-template igxCellEditor let-cell="cell" let-value>
                    <igx-combo type="line" [(ngModel)]="cell.editValue" **[placeholder]="'Locations'"** 
                    [displayKey]="'shop'" [data]="locations" width="220px" [igxFocus]="true" [overlaySettings]="customOverlaySettings"></igx-combo>
                </ng-template>
            </igx-column>`
